Makefile

```diff
@@ -7,13 +7,10 @@ all: build
 clean:
 	cargo clean
 
-.PHONY: start, start-relay, start-frequency, start-frequency-docker, start-manual, start-interval, start-interval-short, start-with-offchain, start-frequency-with-offchain, start-manual-with-offchain, start-interval-with-offchain
```

Comment:
I think this was my fault early on, but Makefile `.PHONY` is space separated not `, ` separated